ui/plugins/eu.esdihumboldt.hale.ui/src/eu/esdihumboldt/hale/ui/service/project/internal/ProjectServiceImpl.java

```diff
@@ -1018,6 +1021,21 @@ public void run(IProgressMonitor monitor)
 				InstanceService is = PlatformUI.getWorkbench().getService(InstanceService.class);
 				is.dropInstances();
 
+				// inform instance processors of the drop
+
+				// XXX This could be done in a more general way by listening to
+				// datasetChanged events from InstanceService
```

Comment:
Yes, I think so to. We only need to find a place where the corresponding `InstanceServiceListener` may "live".